qucs-doc/tutorial/getstarted/content.tex

```diff
@@ -2,7 +2,7 @@
 % Tutorial -- Getting Started with Qucs
 %
 % Copyright (C) 2007 Stefan Jahn <stefan@lkcc.org>
-% Copyright (C) 2007 Juan Carlos Borr�s <jcborras@gmail.com>
+% Copyright (C) 2007 Juan Carlos Borr�s <jcborras@gmail.com>
```

Comment:
I am not sure about the character coding in tex files, but this change should be reversed.